app/build.gradle.kts

```diff
@@ -0,0 +1,255 @@
+    implementation(Depends.Room.runtime)
+    implementation(Depends.Room.rxjava2)
+    kapt(Depends.Room.compiler)
+
+    implementation(Depends.RxJava2.core)
+    implementation(Depends.RxJava2.android)
+    implementation(Depends.RxJava2.kotlin)
+    implementation(Depends.rxbroadcast)
+
+    kapt(Depends.Binding.compiler)
+
+    implementation(Depends.Dagger.core)
+    implementation(Depends.Dagger.android)
+    implementation(Depends.Dagger.androidSupport)
+    kapt(Depends.Dagger.compiler)
+    kapt(Depends.Dagger.androidProcessor)
+
+    implementation(Depends.PlayService.map)
+
+    implementation(Depends.Firebase.firestore)
+    implementation(Depends.Firebase.auth)
+    implementation(Depends.Firebase.core)
+    implementation(Depends.Firebase.messaging)
+
+    implementation(Depends.threetenabp)
+
+    implementation(Depends.Kotpref.kotpref)
+    implementation(Depends.Kotpref.initializer)
+    implementation(Depends.Kotpref.enumSupport)
+
+//==================== UI ====================
+    implementation(Depends.Glide.core)
+    implementation(Depends.Glide.okhttp3)
+    kapt(Depends.Glide.compiler)
+
+    implementation(Depends.Groupie.core)
+    implementation(Depends.Groupie.binding)
+
+    implementation(Depends.downloadableCalligraphy)
+    implementation(Depends.PlayService.oss)
+
+//==================== Debug ====================
+    debugImplementation(Depends.Stetho.core)
+    debugImplementation(Depends.Stetho.okhttp3)
+
+    implementation(Depends.crashlytics) {
+        isTransitive = true
+    }
+
+    implementation(Depends.timber)
+
+    debugImplementation(Depends.leakcanary)
+
+    debugImplementation(Depends.Debot.core)
+    releaseImplementation(Depends.Debot.noop)
+
+//==================== Test ====================
+    testImplementation(Depends.junit)
+    testImplementation(Depends.mockitoKotlin)
+
+    testImplementation(Depends.Robolectric.core)
+    testImplementation(Depends.Robolectric.multidex)
+    testImplementation(Depends.assertk)
+
+    testImplementation(Depends.threetenbp)
+
+    androidTestImplementation(Depends.SupportTest.runner)
+    androidTestImplementation(Depends.SupportTest.espresso)
+    androidTestImplementation(Depends.SupportTest.contrib)
+    androidTestImplementation(Depends.assertk)
+}
+
+repositories {
+    mavenCentral()
+}
+
+play {
+    uploadImages = true
+    setTrack("alpha") // 'production' or 'rollout' or 'beta' or 'alpha'
+    untrackOld = true
+    // userFraction = 0.1
+    jsonFile = file("publisher-keys.json")
+}
+
+ktlint {
+    version = Versions.ktlint
+    android = true
+    reporter = ReporterType.CHECKSTYLE
+    ignoreFailures = true
+}
+
+apply(mapOf("plugin" to "com.google.gms.google-services"))
+
+deploygate {
+    userName = "takahirom"
+    token = System.getenv("DEPLOY_GATE_API_KEY")
+
+    apks {
+        create("release") {
+            val hash = Runtime.getRuntime().exec("git rev-parse --short HEAD")
```

Comment:
hash値がうまく取得できていないような気がします

あまり確認できていないですが下のような感じだと良さそう・・・？

``` Groovy
def hash = 'git rev-parse --short HEAD'.execute([], project.rootDir).in.text.trim()
```
↓
``` Kotlin
val process = ProcessGroovyMethods.execute("git rev-parse --short HEAD", arrayOf<String>(), project.
val hash = ProcessGroovyMethods.getText(process)
println(hash)
```

Reply:
thank you! I will try!

Reply:
The code I wrote was incomplete.
I fixed using Runtime without using Groovy. Because losing if using Groovy.

（Runtimeの処理がすごい途中でした・・・。
ktsにしたのにGroovy使うのは敗北感がすごかったので、Runtimeをこのまま活用してみました。）

```
val hash = Runtime.getRuntime().exec("git rev-parse --short HEAD").inputStream.reader().use { it.readText() }.trim()
```

Reply:
I captured evidence.
<img width="860" alt="2018-02-08 9 25 54" src="https://user-images.githubusercontent.com/73145/35950096-15ede3ba-0cb8-11e8-8953-9d345189d6aa.png">